reconcile-tags/reconcile-tags.py

```diff
@@ -0,0 +1,48 @@
+"""Reads json files mapping docker digests to tags and reconciles them.
```

Comment:
The newest commit should have addressed all of the code comments raised. Tests are forthcoming.